src/util/subprocess.h

```diff
@@ -55,17 +55,13 @@ Documentation for C++ subprocessing library.
 #include <string>
 #include <vector>
 
-#if (defined _MSC_VER) || (defined __MINGW32__)
-  #define __USING_WINDOWS__
-#endif
-
-#ifdef __USING_WINDOWS__
+#ifdef WIN32
   #include <codecvt>
 #endif
 
 extern "C" {
-#ifdef __USING_WINDOWS__
-  #include <Windows.h>
+#ifdef WIN32
+  #include <windows.h>
   #include <io.h>
   #include <cwchar>
 
```

Comment:
Just below here... the `#define` for close/open/fileno breaks using those names in other contexts (eg, C++ stdlib includes, potentially boost, whatever file includes this one, etc). At a minimum, we should probably define them as subprocess_(open|close|fileno) and define those to open/close/fileno for non-Windows too.

Reply:
Thanks! Your commit has been picked.